pkg/install/install_test.go

```diff
@@ -48,38 +48,38 @@ func unmarshalDeployment(t *testing.T, data []byte) *v1.Deployment {
 
 func TestFillInTemplatesAllParameters(t *testing.T) {
 	testFillInTemplates(t, 5, TemplateParameters{
-		GitURL:             "git@github.com:fluxcd/flux-get-started",
-		GitBranch:          "branch",
-		GitPaths:           []string{"dir1", "dir2"},
-		GitLabel:           "label",
-		GitUser:            "User",
-		GitEmail:           "this.is@anemail.com",
-		Namespace:          "flux",
-		GitReadOnly:        false,
-		ManifestGeneration: true,
-		AdditionalFluxArgs: []string{"arg1=foo", "arg2=bar"},
-		RegistryScanning:   true,
-		AddSecurityContext: true,
+		GitURL:                  "git@github.com:fluxcd/flux-get-started",
+		GitBranch:               "branch",
+		GitPaths:                []string{"dir1", "dir2"},
+		GitLabel:                "label",
+		GitUser:                 "User",
+		GitEmail:                "this.is@anemail.com",
+		Namespace:               "flux",
+		GitReadOnly:             false,
+		ManifestGeneration:      true,
+		AdditionalFluxArgs:      []string{"arg1=foo", "arg2=bar"},
+		RegistryDisableScanning: true,
```

Comment:
This should not pass -- you just toggled a flag that's supposed to affect the generated templates!

Reply:
true, it shouldn't. I need to work on re-adding the install package tests later on though